app/client/websocket/index.ts

```diff
@@ -190,6 +190,13 @@ export default class WebSocketClient {
             // reliable websockets are enabled this won't trigger a new sync.
             this.shouldSkipSync = false;
 
+            if (ev.message && typeof ev.message === 'object' && 'code' in ev.message && ev.message.code === 1015) {
+                // code 1015 means an error in the TLS handshake
```

Comment:
Personal opinion - I feel like it's better to have a constant with a meaningful name than a comment to tell us